easybuild/easyconfigs/n/NanoLyse/NanoLyse-1.2.1-foss-2021a.eb

```diff
@@ -0,0 +1,56 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'NanoLyse'
+version = '1.2.1'
+
+homepage = 'https://github.com/wdecoster/nanolyse'
+description = """Remove reads mapping to the lambda phage genome from a fastq file."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake',    '3.20.1'),
+    ('make',     '4.3'),
+    ('Check',    '0.15.2'),
+    ('SWIG',     '4.0.2'),
+    ('expat',    '2.2.9'),
+    ('bzip2',    '1.0.8'),
+    ('zlib',     '1.2.11'),
+]
+
+dependencies = [
+    ('Python',    '3.9.5'),
+    ('Biopython', '1.79'),
+]
+
+use_pip = True
+
+# Fetch the tar.gz, not the whl files!
+exts_list = [
+    ('mappy', '2.24', {
+        'checksums': ['35a2fb73ef14173283d5abb31e7a318429e0330c3be95851df38dd83d4ff9af9'],
+    }),
```

Comment:
I believe `mappy` requires `minimap2`, so that should be a dependency. Given that the `mappy` releases match the `minimap2` releases, I would say the versions should be the same as well

Reply:
There is a `minimap2.py` script in the install bin/

Reply:
will check for minimap2

Reply:
from what I remember, `mappy` just provides a python wrapper that calls out to `minimap2`